File: tests/machine/x/php/match_expr.php
<?php
$x = 2;
$label = (function($_t) {
    if ($_t === 1) return "one";
    if ($_t === 2) return "two";
    if ($_t === 3) return "three";
    return "unknown";
)($x);
var_dump($label);

File: tests/machine/x/zig/match_expr.zig
const std = @import("std");

const x = 2; // i32
const label = switch (x) {1 => "one", 2 => "two", 3 => "three", else => "unknown", }; // []const u8

pub fn main() void {
    std.debug.print("{s}\n", .{label});
}
